winui/Avatar-view/content-types.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn about the available content types and how to set values for them in the WinUI AvatarView control.
+platform: WinUI
+control: AvatarView
+documentation: ug
+---
+
+# Content Types in WinUI AvatarView
+
+The `AvatarView` control supports five different types of content: `Default`, `Initials`, `CustomImage`, `AvatarCharacter`, and `GroupView`.
+
+## Default
+
+Default content type is used for displaying the `Avatar1` character image when initializing the control without the initials or custom image or group source.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView />
+
+{% endhighlight %} 
+{% highlight C# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initials 
+
+The Initials content type is used to display initials in the AvatarView. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. Depending on the specified `InitialsType` property, the initials will be formatted accordingly.
```

Comment:
The Initials content type displays initials in the AvatarView. To use this content type, you must provide a value for the `AvatarName` property, which sets the initials to be displayed in the avatar view. The initials will be formatted accordingly depending on the specified `InitialsType` property.